logic/src/commonMain/kotlin/com/wire/kalium/logic/data/team/TeamRepository.kt

```diff
@@ -32,4 +37,28 @@ internal class TeamDataSource(
             Either.Right(Unit)
         })
     }
+
+    override suspend fun fetchMembersByTeamId(teamId: TeamId, userDomain: String): Either<CoreFailure, Unit> = suspending {
+        wrapApiRequest {
+            teamsApi.getTeamMembers(
+                teamId = teamId,
+                limitTo = null
+            )
+        }.map { teamMemberList ->
+            if (teamMemberList.hasMore.not()) {
```

Comment:
Could you please add a comment here on why this check is made, ignoring the result completely in case the result is paginated?